### PR TITLE
Products data

### PR DESCRIPTION
**Pull request by ahmed-fawzy-metwally**
Add new 2 `sql` files
The first one called `categories.sql` insert some categories to `Database`.
The second one called `products.sql` insert some products to `Database`.
Add 36 image one foreach product.

### CODE REVIEW

File: database/products.sql [hunk 1]
```diff
@@ -0,0 +1,37 @@
+INSERT INTO `Product` (category_id, name, price, image, available) VALUES
+(1, 'Crispy Onion Rings', 6, 'Crispy-Onion-Rings.png', 1),
+(1, 'Ancho Chili Cheese Quesadilla', 6, 'Ancho-Chili-Cheese-Quesadilla.png', 1),
+(1, 'Thai Spring Rolls', 7, 'Thai-Spring-Rolls.png', 1),
+(2, 'Cream of Wild Mushroom', 4, 'Cream-of-Wild-Mushroom.png', 1),
+(2, 'Tasty Chicken Noodle soup', 4, 'Tasty-Chicken-Noodle-soup.png', 1),
```
Comment (Hossam-Tarek) [inline database/products.sql]: The default value for the `available` column is `1` so you don't need to insert it.

**Review by Hossam-Tarek — CHANGES_REQUESTED** [review]
Great job, and awesome images but don't but them in our file `image` so that you don't pollute our project, put them in a different folder that will also contain user's images each in a subfolder one for the product's images and the other for user's images.

File: database/products.sql (hunk 1 above)
Comment (ahmed-fawzy-metwally) [reply database/products.sql]: Done.

Comment (ahmed-fawzy-metwally) [reply database/products.sql]: If subfolder of images not founded that is not polluting our project. but the best is to split them.
but the best done.

**Comment by ahmed-fawzy-metwally** [comment]
All comments are Done.

**Review by Hossam-Tarek — APPROVED** [review]
Well done

**Comment by marwagado** [comment]
good job

**Review by marwagado — APPROVED** [review]

**Pull request merged**